Small problems at using

I had three problems during installation and creating my first invoice:
a) "Full-Fledged Example" from README:

> $ curl -v -X POST \
>     http://localhost:3000/api/invoice/transform-and-create/UBL \
>     -F lang=de
>     -F mapping=@contrib/mappings/default-invoice.yaml \

In line three the backslash is missing

b) https://www.guido-flohr.net/creating-electronic-invoices-with-free-and-open-source-software/
_This will create a directory dist that you can copy wherever you like._
Whenever I move the dist directory, I get an error:
> node:internal/modules/cjs/loader:1137
>   throw err;
>   ^
> 
> Error: Cannot find module '@esgettext/runtime'
> Require stack:
> - /var/www/dist/src/main.js
>     at Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
>     at Module._load (node:internal/modules/cjs/loader:975:27)
>     at Module.require (node:internal/modules/cjs/loader:1225:19)
>     at require (node:internal/modules/helpers:177:18)
>     at Object.<anonymous> (/var/www/dist/src/main.js:3:19)
>     at Module._compile (node:internal/modules/cjs/loader:1356:14)
>     at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
>     at Module.load (node:internal/modules/cjs/loader:1197:32)
>     at Module._load (node:internal/modules/cjs/loader:1013:12)
>     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12) {
>   code: 'MODULE_NOT_FOUND',
>   requireStack: [ '/var/www/dist/src/main.js' ]
> }
> 
> Node.js v18.19.0
> 

Without moving dist it is working, until:
c) changing the format to Factur-X-XRechnung
The software is crashing:

> /var/www/e-invoice-eu-main/dist/src/format/format-factur-x.service.js:239
>         const hash = await crypto.subtle.digest('SHA-512', encoder.encode(data));
>                      ^
> 
> ReferenceError: crypto is not defined
>     at FormatFacturXXRechnungService.setTrailerInfoID (/var/www/e-invoice-eu-main/dist/src/format/format-factur-x.service.js:239:22)
>     at FormatFacturXXRechnungService.createPDFA (/var/www/e-invoice-eu-main/dist/src/format/format-factur-x.service.js:172:14)
>     at FormatFacturXXRechnungService.generate (/var/www/e-invoice-eu-main/dist/src/format/format-factur-x.service.js:88:20)
>     at async InvoiceController.create (/var/www/e-invoice-eu-main/dist/src/invoice/invoice.controller.js:52:30)
> 
> Node.js v18.19.0
> 
After googling the error message, I added before the above line 239 this:
`const crypto = require('crypto');`
Now the software is not crashing anymore and the validator says everthing is correct, but I have no idea, whether that's true.

--- a/src/format/format-factur-x.service.ts
+++ b/src/format/format-factur-x.service.ts
@@ -1,6 +1,6 @@
 import { Textdomain } from '@esgettext/runtime';
 import { Injectable } from '@nestjs/common';
-import crypto from 'crypto';
+import { webcrypto as crypto } from 'crypto';
 import {
 	AFRelationship,
 	PDFArray,

